src/validation.cpp

```diff
@@ -602,6 +605,11 @@ bool MemPoolAccept::PreChecks(ATMPArgs& args, MempoolAcceptResult& result, Works
         return state.Invalid(TxValidationResult::TX_CONFLICT, "txn-already-in-mempool");
     }
 
+    // Check for duplicates in package
```

Comment:
Right good point. I've removed it since checking inputs is a better way to test duplicates/conflicts. I'll note for the future to do context-free sanitization checks before taking the lock :)